charts/internal/seed-controlplane/charts/csi-driver-controller/values.yaml

```diff
@@ -100,4 +100,3 @@ csiSnapshotValidationWebhook:
       memory: 200Mi
   secrets:
     server: csi-snapshot-validation-server
-
```

Comment:
Nit: Please add the newline at the end of the file

Reply:
There were 2 new lines here. Should I revert them?

Reply:
I think that a single newline at the end of the file is enough

Reply:
> There were 2 new lines here. Should I revert them?

I see... Github tends to omit the newlines at the end and hence I got confused. You change is OK